CHANGELOG.md

```diff
@@ -20,8 +20,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#776](https://github.com/MyIntervals/emogrifier/pull/776))
 
 ### Changed
-- Move the code sniffing from Travis CI to GitHub actions
-  ([#832](https://github.com/MyIntervals/emogrifier/pull/832))
+- Move the code quality checks from Travis CI to GitHub actions
+  ([#832](https://github.com/MyIntervals/emogrifier/pull/832),
+  [#833](https://github.com/MyIntervals/emogrifier/pull/833))
```

Comment:
This should be 834.

Reply:
Good catch! I've pushed the correspondinge change.